addons/website_sale/static/src/js/website_sale_utils.js

```diff
@@ -53,12 +53,14 @@ function animateClone($cart, $elem, offsetTop, offsetLeft) {
                     top: $imgtodrag.offset().top,
                     left: $imgtodrag.offset().left
                 })
+                .removeClass('w-100 h-100 h-auto w-auto')
                 .addClass('o_website_sale_animate')
                 .appendTo(document.body)
                 .css({
                     // Keep the same size on cloned img.
                     width: $imgtodrag.width(),
                     height: $imgtodrag.height(),
+                    objectFit: $imgtodrag.css("object-fit")
```

Comment:
Should this be backported ?

Reply:
I think this is related to the new templates being added. I don't seem to find these specific classes on IMGs in 15.0. Maybe I'm wrong?

cc @Brieuc-brd

Reply:
@detrouxdev 
It allows to insure to have a perfect clone of the image for the animation where the image is dragged to the cart. The clone is created when we click on the button and it is removed at the end of the animation.

[https://www.awesomescreenshot.com/video/6771423?key=4d9f534679c0b3e69e6d3618de508303](url)